SU2_CFD/src/solvers/CIncEulerSolver.cpp

```diff
@@ -2878,7 +2875,7 @@ void CIncEulerSolver::GetOutlet_Properties(CGeometry *geometry, CConfig *config,
               AxiFactor = 1.0;
             }
 
-            Density      = V_outlet[nDim+2];
+            Density      = V_outlet[prim_idx.Density()];
```

Comment:
Great to see the replacement of the magical numbers nDim + i with human readable indices